backend/canisters/escrow/impl/src/queries/deposit_account.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{RuntimeState, read_state};
+use canister_api_macros::query;
+use canister_tracing_macros::trace;
+use escrow_canister::deposit_account::{Response::*, *};
+use icrc_ledger_types::icrc1::account::Account;
+
+#[query(candid = true, msgpack = true)]
+#[trace]
+fn deposit_accounts(args: Args) -> Response {
```

Comment:
deposit_account

Reply:
I think this should be something like lookup_swap and return the full swap details. Otherwise there's no way to send someone a swap and for them to validate the swap is what you say it is.
Then as part of that it can return the deposit account.